### ISSUE TITLE
[Feature] Open url

### ISSUE THREAD

**Issue by dw72** [felixse/FluentTerminal#354]
It would be nice to have possibility opening displayed url with Ctrl+Click.

**Comment by AndrewKralovec** [comment]
Looks like xterm has a plugin for this 
xterm/lib/addons/webLinks/webLinks

**Comment by felixse** [comment]
Blocked by https://github.com/xtermjs/xterm.js/issues/2417
